misk/src/test/kotlin/misk/web/JsonForProtoEndpointsTest.kt

```diff
@@ -96,7 +159,37 @@ internal class JsonForProtoEndpointsTest {
             .build()
   }
 
+  class GrpcEchoShipmentToken @Inject constructor() :
+      ShippingGetDestinationWarehouseBlockingServer, WebAction {
+    @Unauthenticated
+    override fun GetDestinationWarehouse(shipment: Shipment): Warehouse {
+      return Warehouse.Builder()
+          .warehouse_token(shipment.shipment_token)
+          .build()
+    }
+  }
+
+  // TODO(jwilson): get Wire to generate this interface.
```

Comment:
so at the moment you have to manually define these if you want to use http/json?

Reply:
no definitely not. I just need to change how Misk generates its test .proto files from that Bash script to the Wire plugin. Seemed like too much for this PR.